pkg/middleware/presigned_url_test.go

```diff
@@ -0,0 +1,121 @@
+package middleware
+
+import (
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+func TestIsSignedRequestWithSignatureParam(t *testing.T) {
+	r := httptest.NewRequest("GET", "https://example.com/example.jpg?OC-Signature=something", nil)
+	if !isSignedRequest(r) {
+		t.Fail()
```

Comment:
wouldn't it be better to have an assertion here? if the test fails we'd rather know why. We can assert on the bool return value 🤷‍♂️